source/common/http/conn_pool_grid.cc

```diff
@@ -332,7 +332,7 @@ ConnectivityGrid::ConnectivityGrid(
   std::chrono::milliseconds rtt =
       std::chrono::duration_cast<std::chrono::milliseconds>(alternate_protocols_->getSrtt(origin_));
   if (rtt.count() != 0) {
-    next_attempt_duration_ = std::chrono::milliseconds(rtt.count() * 2);
+    next_attempt_duration_ = std::chrono::duration_cast<std::chrono::milliseconds>(rtt * 1.5);
```

Comment:
I'm surprised this doesn't break any unit test

Reply:
It did break a test, I just fixed it and pushed a new commit :)